### ISSUE TITLE
Conquest weapons not adding attack properly

### ISSUE THREAD

**Issue by archfiendrubicante** [EdenServer/community#1651]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

### Details
Currently when equipping something like a Windurstan Kukri, you should be receiving a +5 boost to attack in areas where your nation isn't in control of an area. However, this bonus is not currently being calculated correctly.

The below is a /checkparam of two different setups using a single windurstan kukri, and two windurstan kukri. At the time of this image I was standing in Kolshushu, which was under San D'Orian control. Therefore in one instance I should have +10 attack, and in the other, +5.

The bonus isn't being calculated at all. It reflects the text that the regional warper stated, "This area is under windurstan control" even when it isn't. The line of code that determines whether or not an area is under your home nations control doesn't seem to be triggering for these weapons that have additional effects.

Version 30181205_0

![image](https://user-images.githubusercontent.com/10780696/57874733-43c66a80-77df-11e9-9e3e-0be38644833a.png)

**Comment by RottenUnagi** [comment]
@archfiendrubicante Fellow Windurstian here. I just tried this out in Kolshushu/Buburimu (owned by Sandy) with a Combat Caster's Boomerang +1 (same conditions: crafted from a Windurst conquest item; bonus [accuracy+4 in this case] granted "outside own nation's control"). I did see the +4 accuracy bonus in /checkparam after I equipped it.

Are you sure you had signet on at the time you tested it? Signet is required to get the conquest item/conquest area equipment bonuses. I only ask because I almost submitted a bug ticket when I wasn't getting the +MP bonus from Tortoise Shield that I *was* getting just a few hours prior, until I realized what had changed ~ my signet had worn off.
